platform/admin/core/admin-core-jolokia/src/main/java/org/codice/ddf/admin/jolokia/CustomJolokiaServlet.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.jolokia;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.Writer;
+import java.util.Map;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletResponseWrapper;
+import org.boon.Boon;
+import org.jolokia.osgi.servlet.JolokiaServlet;
+import org.jolokia.restrictor.Restrictor;
+import org.osgi.framework.BundleContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@SuppressWarnings("squid:S1989" /* Exceptions caught higher up the stack */)
+public class CustomJolokiaServlet extends JolokiaServlet {
+  private static final Logger LOGGER = LoggerFactory.getLogger(CustomJolokiaServlet.class);
+
+  private static final String STATUS = "status";
+
+  private static final String STACKTRACE = "stacktrace";
+
+  public CustomJolokiaServlet() {
+    super();
+  }
+
+  public CustomJolokiaServlet(BundleContext pContext) {
+    super(pContext);
+  }
+
+  public CustomJolokiaServlet(BundleContext pContext, Restrictor pRestrictor) {
+    super(pContext, pRestrictor);
+  }
+
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+      throws ServletException, IOException {
+    CopyPrintWriter writer = new CopyPrintWriter(resp.getWriter());
+    super.doGet(req, new CopyableHttpServletResponse(resp, writer));
+
+    logErrors(writer);
+  }
+
+  @Override
+  protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+      throws ServletException, IOException {
+    CopyPrintWriter writer = new CopyPrintWriter(resp.getWriter());
+    super.doPost(req, new CopyableHttpServletResponse(resp, writer));
+
+    logErrors(writer);
+  }
+
+  private static void logErrors(CopyPrintWriter writer) {
+    Map jolokiaJson = Boon.fromJson(writer.getCopy(), Map.class);
+    Integer status = (Integer) jolokiaJson.get(STATUS);
+    String stacktrace = (String) jolokiaJson.get(STACKTRACE);
+
+    if (status >= 400 && status < 600 && stacktrace != null) {
+      LOGGER.error(stacktrace);
```

Comment:
This could really fill up the logs at error. Probably better to have this at debug.

Reply:
This will mean no error logs for feature wiring issues.